### ISSUE TITLE
Inconsistent keyword color in PHP

### ISSUE THREAD

**Issue by JPry** [StylishThemes/GitHub-Dark#975]
I noticed when viewing PHP files in GitHub that there's an inconsistent color applied to some of the elements. Here's an example, where one class is extending another:

```php
<?php

class Foo extends Bar
{

}
```

In this case, the word `Bar` should have the same color as `Foo`. When the custom theme is turned off, you can see that GitHub natively colors these elements the same.

I specifically noticed this with the "Monokai – Spacegray Eighties" theme.

Looking at the HTML, it seems that elements with a class of `pl-e` should be styled the same as elements with a class of `pl-en`.

* **Browser**: Safari 12.1.2
* **Operating System**: MacOS 10.14.6
* **Link to page with the issue**: (see inline code block above)
* **Screenshot**:

<img width="691" alt="Screen Shot on 2019-08-06 at 09-36-23" src="https://user-images.githubusercontent.com/871924/62544424-bd458600-b82d-11e9-9c55-4f4318a36843.png">

**Comment by the-j0k3r** [comment]
Hi @JPry thank you for alerting us to this issue.

Afaict its the GtHub Syntax themes issue, almost all handle this differently.
If you switch to using GitHub Dark as a GitHub Syntax theme you will see what you expect.

![Screenshot from 2019-08-06 17-01-38](https://user-images.githubusercontent.com/31389848/62551229-e8e76100-b86b-11e9-931c-a584cf430144.png)

Essentially this means All GitHub Syntax Themes should declare same properties for `.pl-en` and `.pl-e` and they dont, so we need to update all themes that dont comply.

Ill ask @Mottie @xt0rted whats the norm in this case comparing to outside GitHub Syntax Themes before taking action, but since themes are many @JPry feel free to also submit a PR. It should get things started.

Thanks.

**Comment by Mottie** [comment]
GitHub's syntax highlighter is called PrettyLights. It's not public domain, but you can create themes for it using their [theme generator](https://github.com/primer/github-syntax-theme-generator).

We can't change the class names that are applied, but we can change the way they are styled. If you find a theme that needs improvement, especially for a specific language, let us know or open a PR.

**Comment by the-j0k3r** [comment]
I updated the themes to have same colors on pl-e and pl-en which fixes this and also made pl-k a different color in some themes to pl-e and pl-en (around 6 themes were different,

Als0 made all the HEX colors lowercase.

I also fixed a couple of issues around, but when ppl start realizing that all the remainder classes/properties also dont match from theme to theme, well, Im not touching that with a 10 foot pole. hahaha

What needs doing is grab a theme that has all the classes in use and make all other themes the same, so removing unused classes or extra classes and then try to make all themes classes properties consistent across dif themes. Two huge jobs none wants to do and certainly not me either.

Well until next complaint is open here we are.
